PWGHF/D2H/Macros/ML/train_models.py

```diff
@@ -37,11 +37,15 @@
     from hipe4ml import plot_utils
     from hipe4ml.model_handler import ModelHandler
     from hipe4ml.tree_handler import TreeHandler
-    from hipe4ml_converter.h4ml_converter import H4MLConverter
     from sklearn.model_selection import train_test_split
```

Comment:
This line can be moved outside the try except

Reply:
Thanks for the comments! I moved the mentioned dependencies after `hipe4ml` import as advised.